docs/source/overview/cache_groups.rst

```diff
@@ -31,4 +49,273 @@ All servers have to have a :term:`Physical Location`, which defines their geogra
+
+.. _cache-group-servers:
+
+Servers
+-------
+The primary purpose of a Cache Group is to contain servers. In most scenarios it is implied or assumed that the servers are :term:`cache servers`, but this is not required and in fact it is not by any means uncommon for the contained servers to be of some other arbitrary :term:`Type`.
+
+A Cache Group can have zero or more assigned servers, and each server can belong to at most one Cache Group.
+
+.. _cache-group-short-name:
+
+Short Name
+----------
+This is typically an abbreviation of the Cache Group's Name_. The main difference is that it isn't required to be unique.
+
+.. _cache-group-type:
+
+Type
+----
+A Cache Group's :term:`Type` determines what kind of servers it contains. Note that there's no real restriction on the kinds of servers that a Cache Group can contain, and this :term:`Type` serves as more of a guide in certain contexts. The :term:`Types` available by default are described in this section.
+
+.. tip:: Because :term:`Types` are mutable, the actual :term:`Types` that can describe Cache Groups cannot be completely or precisely defined. However, there are no good reasons of which this author can think to modify or delete the default :term:`Types` herein described, and honestly the good reasons to even merely add to their ranks are likely few.
+
+.. table:: Aliases
+
+	+-----------+-----------------------------------------------------------------------------+----------------------------------------------+
+	| Name      | Use(s)                                                                      | Type(s)                                      |
+	+===========+=============================================================================+==============================================+
+	| Type ID   | Traffic Ops client and server Go code, :ref:`to-api` requests and responses | positive integer (``int``, ``bigint``, etc.) |
+	+-----------+-----------------------------------------------------------------------------+----------------------------------------------+
+	| Type Name | :ref:`to-api` requests and responses, Traffic Ops database                  | unchanged (``string``, ``String`` etc.)      |
+	+-----------+-----------------------------------------------------------------------------+----------------------------------------------+
+
+
+EDGE_LOC
+""""""""
+This :term:`Type` of Cache Group contains :term:`Edge-tier cache servers`.
+
+MID_LOC
+"""""""
+This :term:`Type` of Cache Group contains :term:`Mid-tier cache servers`
+
+ORG_LOC
+"""""""
+This :term:`Type` of Cache Group contains :term:`origins`. The primary purpose of these is to group :term:`origins` for the purposes of "multi-site-origins", and it's suggested that if that doesn't meet your use-case that these be mostly avoided. In general, it's not strictly necessary to create :term:`origin` *servers* in ATC at all, unless you have to support "multi-site-origins".
```

Comment:
i don't suppose there is any documentation regarding msos that you can link to

Reply:
There's a guide on setting them up, that's what most things link to. It doesn't feel totally appropriate when you just wanna know what it is, but it's all we got.

Reply:
i think a link to the setup guide makes sense probably.

Reply:
right, that's in the seealso box right under that paragraph

Reply:
oh ok. i missed that.